src/scss/index.scss

```diff
@@ -34,5 +39,22 @@ img {
   .image-preview-wrapper {
     border: 2px solid $preview-image-border;
     min-height: 420px;
+    width: 100%;
   }
 }
+
+#image-preview {
+}
+
+.dz-default {
+  color: #000000;
+}
+
+.container {
+  margin-top: 0;
+  margin-top: 15px;
+}
+
+.top-buffer {
+  margin-top: 25%;
+}
```

Comment:
Please add newlines to your files. You can research more here:

* https://help.github.com/articles/dealing-with-line-endings
* https://blog.codinghorror.com/the-great-newline-schism/